easycheck/easycheck.py

```diff
@@ -758,7 +758,7 @@ def check_argument(
             the default text 'argument'
         expected_type (type, Iterable[type]): the expected type of the item
```

Comment:
Just noticed that in the case of most of the arguments explained in the docstring (except for `handle_with` and `**kwargs`), their types are incorrect. They should be the very same as those in the argument list in the function's signature. Please update these types in the docstring.

Reply:
done